docs/starting-out/your-first-server.md

```diff
@@ -1,16 +1,17 @@
 ---
-description: Includes everything you need to have your first server running.
+description: Describes how to get a basic server running.
 ---
 
 # Your first server
 
-Some things are needed before being able to connect to your Minestom server.
+There are a few steps necessary before you can connect to your Minestom server:
 
-- Initialize the server
-- Registering events/commands
-- Start the server at the specified port and address
+- Initializing the server
+- Registering your events/commands
+- Creating an instance for players to join
+- Starting the server on a port and address
 
-Here is a correct example:
+Here is an example of a server without an instance:
```

Comment:
It seems unnecessary to have two separate sections for a wrong one and then a correct one. Can prob just be merged together.